website/docs/api/admin/user-admin.md

```diff
@@ -115,7 +115,17 @@ You can also search for users via the search API. It will preform a simple searc
 
 `POST https://unleash.host.com/api/admin/user-admin`
 
-Creates a new use with the given root role.
+Creates a new user with the given root role.
+
+**Payload properties**
+
+| Property name | Required | Description                                                                     | Example value(s)       |
+|---------------|----------|---------------------------------------------------------------------------------|------------------------|
+| `email`       | Yes      | The user's email address.                                                       | `"user@getunleash.io"` |
+| `name`        | Yes      | The user's name                                                                 | `"Some Name"`          |
+| `rootRole`    | Yes      | The role to assign to the user. Can be either the role's ID or its unique name. | `2`, `"Editor"`        |
```

Comment:
Are all of these fields required?

Reply:
@chriswk Sorry it's taken so long. This kind of fell by the wayside. Reading the code for this, it seems that `rootRole` is required. `email` and `name` both seem to be optional, but you _must_ specify at least one of them. Is that correct?

Reply:
I think you're right, you need at least one, if you want the user to be able to login I believe they'll need an email, since that's what we put the uniqueness on. So we should probably update the endpoint to require email

Reply:
Thank you! ❤️  Updated 🤓